.github/workflows/kotlin.yml

```diff
@@ -1,10 +1,11 @@
 on:
-  push:
-    branches:
-      - master
-  pull_request: {}
+    push:
+      # Pattern matched against refs/tags
+      tags:        
```

Comment:
this prevent the job be run until there is a tag? Can we have an `if: tag` in the publish step instead?

Reply:
updated on 081d41b3b2a3229531084c34c1186b6a423aae53. 
Could you review the condition to trigger the publishing?